dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKRunner.java

```diff
@@ -123,6 +118,12 @@ public class TCKRunner {
     private boolean isTestNG = false;
     private File tckRunnerDir = new File("publish/tckRunner").getAbsoluteFile();
     private File loggingPropertiesFile = new File(tckRunnerDir, "logging.properties");
+    
+    // Default timeout and boolean settings
+    private Boolean failSafeUndeployment = Boolean.TRUE;
+    private Duration appDeployTimeout = Duration.ofSeconds(180);
+    private Duration appUndeployTimeout = Duration.ofSeconds(60);
+    private Duration mbeanTimeout = Duration.ofMillis(60000);
```

Comment:
```suggestion
    private Duration mbeanTimeout = Duration.ofSeconds(60);
```
suggestion: use a duration that is more human readable